madagascar-conservation-and-development.csl

```diff
@@ -0,0 +1,338 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" et-al-min="4" et-al-use-first="1" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Madagascar Conservation &amp; Development</title>
+    <title-short>Madag. conserv. dev.</title-short>
+    <id>http://www.zotero.org/styles/madagascar-conservation-and-development</id>
+    <link rel="self" href="http://www.zotero.org/styles/madagascar-conservation-and-development"/>
+    <link href="http://journalmcd.com/mcd/index.php/mcd/about/submissions#authorGuidelines" rel="documentation"/>
+    <author>
+      <name>Jordi Salmona</name>
+      <email>jordi.salmona@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="conservation"/>
```

Comment:
fields are limited to the list here: http://docs.citationstyles.org/en/latest/specification.html#appendix-i-categories -- you could do biology and social_science or so.